src/Actions/Account/ListAccounts.php

```diff
@@ -0,0 +1,81 @@
+<?php
+
+namespace Choredo\Actions\Account;
+
+use Choredo\Actions\Behaviors;
+use Choredo\Entities\Account;
+use Choredo\EntityManagerAware;
+use Choredo\HasEntityManager;
+use Choredo\Output;
+use Choredo\Transformers\AccountTransformer;
+use Psr\Http\Message\ResponseInterface as Response;
+use Psr\Http\Message\ServerRequestInterface as Request;
+use Zend\Diactoros\Response\JsonResponse;
+use const Choredo\REQUEST_FAMILY;
+use const Choredo\REQUEST_FILTER;
+use const Choredo\REQUEST_PAGINATION;
+use const Choredo\REQUEST_SORT;
+
+class ListAccounts implements
+    EntityManagerAware,
+    Output\FractalAwareInterface,
+    Behaviors\Filterable,
+    Behaviors\Pageable,
+    Behaviors\Sortable
+{
+    use Output\CreatesFractalScope;
+    use HasEntityManager;
+    use Behaviors\HasDefaultCreateDateSort;
+    use Behaviors\HasDefaultPaginationLimits;
+    use Behaviors\HasNoFilterTransforms;
+
+    /**
+     * Return an array of fields that can be filtered via the API
+     *
+     * @return string[]
+     */
+    public static function getFilterableFields(): array
+    {
+        return ['emailAddress', 'firstName', 'lastName'];
+    }
+
+    /**
+     * @return string[]
+     */
+    public static function getSortableFields(): array
+    {
+        return ['createdDate', 'lastLogin'];
```

Comment:
Feels like `firstName` and `lastName` would be valuable sort fields too, no?

Reply:
:+1: